osi_object.proto

```diff
@@ -54,6 +54,15 @@ message Vehicle
     /// The id of the driver of the (ego) vehicle.
     /// Field need not be set if ego_vehicle is set to false.
     optional Identifier driver = 10;
+    
+    /// The vector pointing from the bounding box center point (base.position) to the middle (in x, y and z) 
+    /// of the rear axle under neutral load conditions.
+    optional Vector3d bbcenter_to_rear = 11;
+
+	/// The vector pointing from the bounding box center point (base.position) to the middle (in x, y and z) 
+	/// of the front axle under neutral load conditions.    		
+	optional Vector3d bbcenter_to_front = 12;
+    
 
```

Comment:
Replacing relative_ego_origin is fine, especially since it is only a rename and the addition of the front axis coordinate system now.
The (only) purpose of relative_ego_origin is/was to allow the transformation to a frame relative to the rear_axis_coordinate system of a vehicle (specifically ego-vehicles, that's why it wasn't required for all vehicles). That is still possible with the new definitions, so everything is perfectly fine there.